rosdep/python.yaml

```diff
@@ -4455,6 +4455,7 @@ python-validictory-pip:
       packages: [validictory]
 python-vcstool:
   debian:
+    bookworm: [vcstool]
```

Comment:
This will install the Python 3 vcstool package, but this key targets (unversioned) Python 2 vcstool. Python 2 vcstool isn't available for debian bookworm.

Reply:
> This will install the Python 3 vcstool package, but this key targets (unversioned) Python 2 vcstool. Python 2 vcstool isn't available for debian bookworm.

fixed in latest force push